src/python/test/test_nextflow.py

```diff
@@ -54,6 +54,18 @@
     "help": "second input",
     "label": "Test2"
 }
+input3 = {
+    "class": "file",
+    "name": "third_input",
+    "help": "(Nextflow pipeline optional)third input",
```

Comment:
I see that help message of the newly added inputs contain ()Nextflow pipeline optional), but the help message of input1/2 does not. Shouldnt they be the same?

Reply:
I kept the previous ones in case it is needed in some other tests you are using. But let me know if those are not needed anymore. 

I think we can keep all, although currently the only case that we have custom inputs is when using schema, and the inputs will have help message either with (Nextflow pipeline optional) or Nextflow pipeline required). However if later we allow getting more customizable inputs, input1/2 will cover those cases. cc @commandlinegirl